src/object.js

```diff
@@ -345,6 +351,14 @@ Sk.builtin.object.prototype["__ge__"] = function (self, other) {
     return self.ob$ge(other);
 };
 
+Sk.builtin.object.prototype.__get__ = function __get__(self, instance, owner) {
+    Sk.builtin.pyCheckArgs("__get__", arguments, 1, 2, false, true);
+
+    var l_owner = !owner ? instance.ob$type : owner;
+
+    return self.tp$descr_get(instance, l_owner);
```

Comment:
And what's the default impl of `tp$descr_get()`? (I might need to just read this more closely...)

Reply:
Good question ... I don't think this should be here to be honest. Because object doesn't have __get__ in python2 right?

Reply:
Just checked...no, they don't! (Having `__get__` would turn _every_ object into a descriptor, which is probably not what you want!)